cumulusci/robotframework/tests/salesforce/faker.robot

```diff
@@ -0,0 +1,67 @@
+*** Settings ***
+Resource  cumulusci/robotframework/Salesforce.robot
```

Comment:
This suite should probably use `Force Tags  no-browser` so it runs in the CI job for tests that don't require a browser.

Reply:
Done.